cmd/posix.go

```diff
@@ -72,7 +72,7 @@ func checkPathLength(pathName string) error {
 	return nil
 }
 
-func checkPathValid(path string) (string, error) {
+func getValidPath(path string) (string, error) {
```

Comment:
Looks like all that this function does is to check if the path is valid. So, will `checkPathValid` be a better name? Also, the input path is returned again as is, can the return be just an error?